scout/adapter/mongo/case.py

```diff
@@ -112,21 +112,21 @@ def _set_genes_of_interest_query(self, query, query_field, query_term):
             query_field(str) "pinned" or "causative"
             query_term(str) example:"POT1"
         """
+        LOG.warning(f"IN GENES OF INTEREST--->{query_field}:{query_term}")
         hgnc_id = self.hgnc_id(hgnc_symbol=query_term)
         if hgnc_id is None:
             LOG.debug(f"No gene with the HGNC symbol {query_term} found.")
             query["_id"] = {"$in": []}  # No result should be returned by query
+            return
 
         unwind = "$causatives"
         lookup_local = "causatives"
-        lookups_as = "causative_variant"
-        match = "causative_variant.hgnc_ids"
-
         if query_field == "pinned":
             unwind = "$suspects"
             lookup_local = "suspects"
-            lookups_as = "suspect_variant"
-            match = "suspect_variant.hgnc_ids.hgnc_ids"
```

Comment:
This typo is the cause of the bug; `hgnc_ids.hgnc_ids ` instead of just `hgnc_ids`

Reply:
Did `suspect_variant` collide somewhere else or why `lookedup_variant`? 😊

Reply:
It's just to give it a common name that can be used by both suspects or pinned. We can can rename it something else if you don't like it.

Reply:
Hehe, no, by all means! It's just very deep in the loop layers so to speak. If I read it right, it's like a variable for what the mongodb query temp name will be. We won't really do anything more with it, more than snarf all the case_ids returned, so it might just as well be a literal constant in the query? 😊

Reply:
Yes, you can name it whatever you want, it's just a temp variable used when running the query, I'll name it as a constant!